### PR TITLE
remove cloud simulator doc pages

### LINKED ISSUE #1424: Remove cloud simulator docs
Closes #607. 

Remove /verify/using-ibm-quantum-simulators and /verify/cloud-based-simulators, now that cloud-based simulators have been sunset. Also search the docs and remove any links to or mention of these docs and the cloud simulators. For example, in the first paragraph of the migration guide (/api/migration-guides/local-simulators), there is a link to the available cloud-based sims page.

### PR DESCRIPTION
**Pull request by abbycross**
Closes #1424 and #607

### CODE REVIEW

**Comment by abbycross** [comment]
@beckykd could you please remove the example with the qasm simulator on this page?: https://docs.quantum.ibm.com/api/migration-guides/qiskit-runtime-examples -- you can use this branch if it makes it easier!

**Review by Eric-Arellano — COMMENTED** [review]
Do we know what the 404 plan is? Redirect to something like a product announcement? Please be careful to merge this until we have that figured out because the next time we do an open source sync would cause 404s.

**Comment by beckykd** [comment]
> @beckykd could you please remove the example with the qasm simulator on these migration guide pages?:
> 
> https://docs.quantum.ibm.com/api/migration-guides/qiskit-runtime-examples https://docs.quantum.ibm.com/api/migration-guides/qiskit-algorithms-module
> 
> you can use this branch if it makes it easier!

I found another one in https://docs.quantum.ibm.com/build/operators-overview -- can we just change that to a different backend?  Maybe @kaelynj knows.

**Comment by beckykd** [comment]
> @beckykd could you please remove the example with the qasm simulator on these migration guide pages?:
> 
> https://docs.quantum.ibm.com/api/migration-guides/qiskit-runtime-examples https://docs.quantum.ibm.com/api/migration-guides/qiskit-algorithms-module
> 
> you can use this branch if it makes it easier!

**qiskit-runtime-examples**: all uses are in the legacy/backend.run() tabs, so I think they're OK.

**qiskit-algorithms-module** is more tricky.  It's part of the Qiskit.44 changes -- so none of it is updated to use V2 primitives and the code won't work anymore anyway.  Do we still need the Qiskit .44 migration guides?  As a stopgap, I did update the "updated" example in the **VQD Example** section to run on a real backend but left the Legacy example running on qasm_simulator.   Still, it's not going to run without deprecation warnings; and in a month or so (?) it won't work at all.

File: docs/api/migration-guides/local-simulators.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: Migrate from using the IBM Quantum cloud simulators to using Qiskit
 
 # Migrate from cloud simulators to local simulators
 
-In quantum computing, the choice between using simulators and quantum hardware is crucial for making progress in the field. While simulators are useful for testing and debugging, in this era of quantum utility, quantum development and industry advancement requires actual hardware. As part of the move to quantum utility, IBM Quantum&trade; will retire its cloud simulators on 15 May 2024. This guide explains in more detail why this is happening, and how to migrate from [cloud based simulators](/verify/cloud-based-simulators), such as `ibmq_qasm_simulator`, to local simulators.
+In quantum computing, the choice between using simulators and quantum hardware is crucial for making progress in the field. While simulators are useful for testing and debugging, in this era of quantum utility, quantum development and industry advancement requires actual hardware. As part of the move to quantum utility, IBM Quantum&trade; will retire its cloud simulators on 15 May 2024. This guide explains in more detail why this is happening, and how to migrate from cloud-based simulators, such as `ibmq_qasm_simulator`, to local simulators.
```
Comment (Eric-Arellano) [inline docs/api/migration-guides/local-simulators.mdx:8]: Should this use past tense?

**Review by Eric-Arellano — APPROVED** [review]

File: docs/api/migration-guides/local-simulators.mdx (hunk 1 above)
Comment (abbycross) [inline docs/api/migration-guides/local-simulators.mdx:8]:
```suggestion
In quantum computing, the choice between using simulators and quantum hardware is crucial for making progress in the field. While simulators are useful for testing and debugging, in this era of quantum utility, quantum development and industry advancement requires actual hardware. As part of the move to quantum utility, IBM Quantum&trade; cloud simulators were retired on 15 May 2024. This guide explains the retirement in more detail, and how to migrate from cloud-based simulators, such as `ibmq_qasm_simulator`, to local simulators.
```

**Pull request merged**